### ISSUE TITLE
[Bug] Accessibility Help missing

### ISSUE THREAD

**Issue by Stopa** [microsoft/monaco-editor#4908]
### Reproducible in vscode.dev or in VS Code Desktop?

- [x] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Link

https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0#example-creating-the-editor-hello-world

### Monaco Editor Playground Code

```typescript
const value = /* set from `myEditor.getModel()`: */ `function hello() {
	alert('Hello world!');
}`;

// Hover on each property to see its docs!
const myEditor = monaco.editor.create(document.getElementById("container"), {
	value,
	language: "javascript",
	automaticLayout: true,
	accessibilityHelpUrl: 'github.com'
});
```

### Reproduction Steps

A:
1. Open the command palette (F1)
2. search for "accessibility"

B:

1. Press Alt+F1 (open accessibility help)

### Actual (Problematic) Behavior

A:

- Nothing is available

B:

- Nothing happens

### Expected Behavior

A:

- "Show Accessibility Help" option is available

B:

- Accessibility Help pop-over opens

### Additional Context

As a result on https://github.com/microsoft/vscode/issues/179689#issuecomment-2883984545 , stand-alone monaco editor on the web no longer has the accessibility help dialog. It is not accessible through the previously used Alt+F1 shortcut nor is the "Show Accessibility Help" command available in the command palette.

And since the dialog is missing, the `accessibilityHelpUrl` configuration parameter (https://microsoft.github.io/monaco-editor/docs.html#interfaces/editor.IStandaloneEditorConstructionOptions.html#accessibilityHelpUrl) is also not usable.

This makes the stand-alone monaco editor less accessible, as now there is no way for users to discover the screen reader optimizations or switch out Tab key effect (`Control+Shift+M` shortcut that still works).

I see that the information about Accessibility Help menu has been removed from the monaco editor accessibility wiki page (https://github.com/microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide/_compare/acf6ee45faa4a210c55cba7d8373ba5b7b185b69...bc5d5974a5e422178c4b0cb4afe9850393f11635), but it's still mentioned in the Accessibility Guide for Integrators (https://github.com/microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators).

Is there a new recommended approach to add accessibility help in the use-case of stand-alone monaco editor (non-vscode) integration?

**Comment by jruales** [comment]
Any updates on this?

Just to add to the above, it seems like Accessibility Help is gone from monaco-editor version 0.40.0 onwards.

**Comment by jruales** [comment]
See also: https://github.com/microsoft/monaco-editor/issues/4124
